wsd/COOLWSD.cpp

```diff
@@ -1199,6 +1199,23 @@ void COOLWSD::innerInitialize(Poco::Util::Application& self)
         { "logging.docstats", "false" },
         { "logging.userstats", "false" },
         { "logging.disable_server_audit", "false" },
+        { "logging_ui_cmd.file.property[0]", "coolwsd-ui-cmd.log" },
+        { "logging_ui_cmd.file.property[0][@name]", "path" },
+        { "logging_ui_cmd.file.property[1]", "never" },
```

Comment:
I think all these options like rotation and archive etc aren't needed, they are complicated things that the "main log" has to compress and backup these logs etc, and we probably don't need them for this log and it would makes things smaller and simpler to not have them. We could add them if we need them later.

Reply:
I deleted some from coolwsd.xml.in, but i left default values here.
If user set them, Poco will use them.

I also added 2 non Poco setting .. so it can make unümerged logs if needed.. and it can write time intervall if needed.